app/src/ingestion/scrapy_dst/spiders/ca_public_charge_spider.py

```diff
@@ -50,7 +51,7 @@ def parse_page(self, response: HtmlResponse) -> dict[str, str | AccordionSection
         # remove icon text
         response.css("div.ic-icon").drop()
 
-        extractions |= self.parse_main_primary(base_url, response.css("div.module-full"))
+        # extractions |= self.parse_module_full(base_url, response.css("div.module-full"))
```

Comment:
The parsing results from `response.css("div.module-full")` is never used, so comment it out.